engine/src/main/java/org/terasology/logic/console/commands/ClientCommands.java

```diff
@@ -36,12 +37,21 @@
     private WorldProvider worldProvider;
 
 
+    /**
+     * Displays debug information on the target entity
+     * @return String
+     */
     @Command(shortDescription = "Displays debug information on the target entity")
     public String debugTarget() {
         EntityRef cameraTarget = cameraTargetSystem.getTarget();
         return cameraTarget.toFullDescription();
     }
 
+    /**
+     * Sets the current world time of the in days fr the local player
```

Comment:
This is again simply copied from the description, including the weird typo. Probably fix both the description and the javadoc.

"Sets the current world time for the local player in days"

Be sure to also have a line for the parameter and return